content/2.components/text-reveal-card.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Text Reveal Card
+description: Mousemove effect to reveal text content at the bottom of the card.
+---
+
+::ComponentLoader{label="Preview" componentName="TextRevealDemo" type="examples"}  
+::
+
+::alert{type="info"}
```

Comment:
This info alert is not needed here since vueuse/motion is mentioned as required dependency in installation guide.